File: platforms/web/examples/scene/camera/bind.js
import { Camera, Material, Renderer } from "fragmentcolor";

const camera = Camera.perspective(60.0.toRadians(), 16.0 / 9.0, 0.1, 100.0).lookAt([0.0, 1.0, 5.0], [0.0, 0.0, 0.0], [0.0, 1.0, 0.0]);

const renderer = new Renderer();
const material = await Material.pbr(renderer);
camera.bind(material.shader());

File: platforms/kotlin/examples/scene/light/bind.kt
import org.fragmentcolor.*

val renderer = Renderer()
val material = Material.pbr(renderer)
val sun = Light.directional(listOf(0.3f, -1.0f, -0.4f), listOf(1.0f, 0.95f, 0.9f))
sun.bind(material.shader())